src/library/scala/collection/generic/IsSeqLike.scala

```diff
@@ -1,44 +1,97 @@
 package scala.collection
 package generic
 
+import scala.reflect.ClassTag
+import scala.language.higherKinds
 
 /** Type class witnessing that a collection representation type `Repr` has
-  * elements of type `A` and has a conversion to `SeqOps[A, Seq, Repr]`.
+  * elements of type `A` and has a conversion to `SeqOps[A, Iterable, C]`, for
+  * some types `A` and `C`.
   *
   * This type enables simple enrichment of `Seq`s with extension methods which
   * can make full use of the mechanics of the Scala collections framework in
   * their implementation.
   *
   * @see [[scala.collection.generic.IsIterableLike]]
   */
-trait IsSeqLike[Repr] {
-  /** The type of elements we can traverse over. */
-  type A
-  /** A conversion from the representation type `Repr` to `SeqOps[A, Seq, Repr]`. */
-  val conversion: Repr => SeqOps[A, Seq, Repr]
+trait IsSeqLike[Repr] extends IsIterableLike[Repr] {
+
+  @deprecated("'conversion' is now a method named 'apply'", "2.13.0")
+  override val conversion: Repr => SeqOps[A, Iterable, C] = apply(_)
+
+  /** A conversion from the type `Repr` to `SeqOps[A, Iterable, C]` */
```

Comment:
maybe add a comment why this doesn't return a `SeqOps[A, Seq, C]` (I guess because of SeqView)? similarly in `IsMapLike`.

Reply:
Done.